components/Cards/DetailsCard.js

```diff
@@ -24,6 +31,105 @@ export default function DetailsCard({ id, isFavorite, toggleFavorite, place }) {
   } = place;
 
   const [comments, setComments] = useState([]);
+  const [editingCommentId, setEditingCommentId] = useState(null);
+  const session = useSession();
+
+  //----------------------------------------------------------------
+
+  const handleEdit = (commentId) => {
+    setEditingCommentId(commentId);
+  };
+
+  //----------------------------------------------------------------
+
+  const handleDelete = async (commentId) => {
+    try {
+      const response = await fetch(`/api/comments/${commentId}`, {
+        method: "DELETE",
+      });
+      if (response.ok) {
+        console.log("Comment deleted successfully");
```

Comment:
Your user usually doesn't see the console.
```suggestion
        window.alert("Comment deleted successfully");
```